src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py

```diff
@@ -472,6 +470,7 @@ def _request(self, retry=0, outputFile=None, **kwargs):
     if self.__extraCredentials:
       kwargs[self.KW_EXTRA_CREDENTIALS] = encode(self.__extraCredentials)
     kwargs["clientVO"] = self.vo
+    # kwargs["clientSetup"] = self.setup
```

Comment:
This is not the solution, it seems. But I am not knowledgeable enough to understand what would be right fix.

Reply:
@chaen the TornadoJobManager is still problematic. This is hotfixed on lbcertifdirac70 but still fails with

```
Traceback (most recent call last):
  File "/opt/dirac/versions/v7.3.0a22-1629435055/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/Tornado/Server/TornadoService.py", line 451, in __executeMethod
    self.initializeRequest()
  File "/opt/dirac/versions/v7.3.0a22-1629435055/Linux-x86_64/lib/python3.9/site-packages/DIRAC/WorkloadManagementSystem/Service/JobManagerHandler.py", line 96, in initializeRequest
    self.diracSetup = self._serviceInfoDict['clientSetup']
KeyError: 'clientSetup'
```
I didn't look much further, I am not sure what is the best solution here.

Reply:
Where can I see the actual problem and how to reproduce it ?